src/main/java/examples/pingpong/reactive/PingPongReactiveMain.java

```diff
@@ -50,130 +49,81 @@ public static void main(String[] args) {
-        PingPongProcessor bobProcessor = new PingPongProcessor(BOB, client, pingIdentifier, pongIdentifier);
+        PingPongProcessor aliceProcessor = new PingPongProcessor(aliceParty, client, pingIdentifier, pongIdentifier);
+        PingPongProcessor bobProcessor = new PingPongProcessor(bobParty, client, pingIdentifier, pongIdentifier);
 
         // start the processors asynchronously
         aliceProcessor.runIndefinitely();
         bobProcessor.runIndefinitely();
 
         // send the initial commands for both parties
-        createInitialContracts(client, ALICE, BOB, pingIdentifier, numInitialContracts);
-        createInitialContracts(client, BOB, ALICE, pingIdentifier, numInitialContracts);
-
+        createInitialContracts(client, aliceParty, bobParty, pingIdentifier.toProto(), numInitialContracts);
+        createInitialContracts(client, bobParty, aliceParty, pingIdentifier.toProto(), numInitialContracts);
 
         try {
             // wait a couple of seconds for the processing to finish
-            Thread.sleep(5000);
+            Thread.sleep(20000);
             System.exit(0);
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
     }
 
     /**
-     * Creates numContracts number of Ping contracts. The sender is used as the submitting party.
+     * Creates numContracts number of Ping contracts. The sender is used as the
+     * submitting party.
      *
      * @param client         the {@link LedgerClient} object to use for services
      * @param sender         the party that sends the initial Ping contract
      * @param receiver       the party that receives the initial Ping contract
      * @param pingIdentifier the PingPong.Ping template identifier
      * @param numContracts   the number of initial contracts to create
      */
-    private static void createInitialContracts(LedgerClient client, String sender, String receiver, Identifier pingIdentifier, int numContracts) {
+    private static void createInitialContracts(LedgerClient client, String sender, String receiver,
+            Identifier pingIdentifier, int numContracts) {
 
         for (int i = 0; i < numContracts; i++) {
-            // command that creates the initial Ping contract with the required parameters according to the model
-            CreateCommand createCommand = new CreateCommand(pingIdentifier,
-                    new DamlRecord(
-                            pingIdentifier,
-                            new DamlRecord.Field("sender", new Party(sender)),
-                            new DamlRecord.Field("receiver", new Party(receiver)),
-                            new DamlRecord.Field("count", new Int64(0))
-                    )
-            );
+            // command that creates the initial Ping contract with the required parameters
+            // according to the model
+
+            Command createCommand = Command.newBuilder().setCreate(
+                    CreateCommand.newBuilder()
```

Comment:
indentation seems a bit weird, not sure if this is autoformatted, maybe with 8-space indents?